gradlew.bat

```diff
@@ -9,7 +9,7 @@
 if "%OS%"=="Windows_NT" setlocal
 
 @rem Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-set DEFAULT_JVM_OPTS=
+set DEFAULT_JVM_OPTS=-Dfile.encoding=UTF-8

```

Comment:
It is better to add a `gradle-properties` file and pass the JVM arg in it:

```
org.gradle.jvmargs='-Dfile.encoding=UTF-8'
```

Reply:
Thank you  for your good advice.
I check the gradle properties and encoding more, and will modify (or rewrite) PR.